### ISSUE TITLE
oncoPlot Fails if Exactly One Sample has Mutations

### ISSUE THREAD

**Issue by DarioS** [PoisonAlien/maftools#206]
In the following example, the MAF has two samples, but only one has any mutations in the categories which maftools considers.

```
library(maftools)
library(TCGAbiolinks)
library(magrittr)

mutations <- GDCquery_Maf("HNSC", pipelines = "muse")
keep <- grep("7382|7391", unlist(mutations[, "Tumor_Sample_Barcode"]))
# 7391 only has one 3' Flank mutation. Nothing to plot for it.
mutations <- mutations[keep, ]
mutationsMAF <- mutations %>% read.maf

oncoplot(mutationsMAF)
```

This results in 

```
Error in createOncoMatrix(m = maf, g = genes) : 
  object 'sampleId' not found
```

**Comment by PoisonAlien** [comment]
Hi,
I have fixed this issue in recent updates (1.8.10). Could you please update and let me know ? Also most of the maftools functions will complain with single sample as it is meant to work with cohorts.

**Comment by DarioS** [comment]
I find that the software now catches this case with a clear error message.
